chore(docs): Attempt to fix docs and bb docs previews #16418

critesjosh opened the pull request:
another attempt at fixing previews since https://github.com/AztecProtocol/aztec-packages/pull/16409 didnt totally work

alejoamiras commented on docs/netlify.toml:
```diff
@@ -1,6 +1,6 @@
 [build]
   # Only build if changes are made in the docs directory
-  ignore = "git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF -- docs"
+  ignore = "git diff --quietorigin/next HEAD -- docs"
```
```suggestion
  ignore = "git diff --quiet origin/next HEAD -- docs"
```

critesjosh replied:
Derp thanks

critesjosh committed:
chore(docs): Attempt to fix docs and bb docs previews #16418

another attempt at fixing previews since https://github.com/AztecProtocol/aztec-packages/pull/16409 didnt totally work
